content/docs/command-reference/import.md

```diff
@@ -74,6 +74,23 @@ Note that import stages are considered always
 they won't be updated. Use `dvc update` to update the downloaded data artifact
 from the source repo.
 
+### Backup mode
+
+To avoid duplicating large files, `dvc import` does not push imported files to
+any [remotes](/doc/command-reference/remote) by default. This means that
+`dvc pull`, `dvc fetch`, and `dvc repro` retrieve imports from their source. So if
+the original source of an import is deleted, then the files are not backed up
+anywhere, and data may be lost.
+
+To prevent this, you can use the `--backup` option to request DVC to backup the
+data imports to remote storage. This also means that `dvc pull` or `dvc fetch`
```

Comment:
> `dvc import` does not push imported files

Will `import` do the pushing when using --backup? Or that just tells `push` to do it when/if run? I'm guessing the latter but it's not clear from the text above (also in the 2nd paragraph). Feel free to update it or just lmk what the answer is and I will 👍

Reply:
Yes, the latter: I'll tweak